app/src/main/java/com/vero/tasky/agenda/presentation/util/AgendaTimeParser.kt

```diff
@@ -0,0 +1,29 @@
+package com.vero.tasky.agenda.presentation.util
+
+import com.vero.tasky.agenda.domain.model.AgendaItem
+import java.time.LocalDateTime
+import java.time.format.TextStyle
+import java.util.*
+
+object AgendaTimeParser {
+
+    fun getTimeLine(agendaItem: AgendaItem) : String {
+        return when (agendaItem) {
+            is AgendaItem.Event -> {
+                val from = getParsedTimeLine(dateTime = agendaItem.time)
+                val to = getParsedTimeLine(dateTime = agendaItem.to)
+                "$from - $to"
```

Comment:
This will only be used on the presentation layer, so it should be moved there

Reply:
What do you mean by that? Currently, it is in the presentation/util